che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/build/StartBuildAction.java

```diff
@@ -48,22 +48,23 @@
     private final OpenshiftServiceClient        openshiftService;
     private final NotificationManager           notificationManager;
     private final OpenshiftLocalizationConstant locale;
-    private final BuildStatusWatcher            buildStatusWatcher;
+
```

Comment:
What is special in BuildsPresenter? Why do you separate it from other fields with empty line?